### PR TITLE
Activate CIP20

### PR DESCRIPTION
**Pull request by harry1453**
This pull request will serve as the comments thread for this CIP.

Comment period is 30 days, from 12/04/2019 to 12/05/2019.

### CODE REVIEW

File: cip-0020.md [hunk 1]
```diff
@@ -22,7 +21,7 @@ The parameters for the AT fee structure are handled in the `AT_Constants.java` f
 
 | Constant | AT version 1 | AT version 2|
 |----------|-------------|-------------|
-| `STEP_FEE` | `10.1 BURST`  | `FEE_QUANT/10 BURST` |
+| `STEP_FEE` | `1 BURST`  | `FEE_QUANT/10 BURST` |
```
Comment (frankTheTank72) [inline cip-0020.md]: Need to be 0.1  BURST not 1

File: cip-0020.md [hunk 2]
```diff
@@ -62,6 +61,10 @@ Unlike other blockchains, BURST does not have its own colored coin for running A
 
 The change will require a hard-fork. Any ATs created before the fork will be subject to the previous set of fees and will operate without any change. Only ATs created after the fork will run under the new fees.
 
+## Additional notes
+
+Upon activation, this CIP will be implemented in BRS and the changes to fee structure will take effect during the next hard fork.
+
```
Comment (frankTheTank72) [inline cip-0020.md]: Can we add a block high from here for in 2 months or so ?

Comment (harry1453) [reply cip-0020.md]: Block height for next fork will be determined once we have laid out the full extent of our plans for that fork and, more importantly, have network consensus.

Comment (frankTheTank72) [reply cip-0020.md]: oaky

**Comment by jjos2372** [comment]
Please consider also increasing MAX_MACHINE_CODE_PAGES from 10 to 20. With availability of BlockTalk, more complex contracts are manageable and the 10 pages limit for machine code is easily reached.

**Comment by harry1453** [comment]
ok @jjos2372

**Pull request merged**